src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,287 @@ paths:
         500:
           description: "Internal Server Error"
 
+  /alert-subscriptions:
+    post:
```

Comment:
The available methods for a given endpoint should be listed in our standard order of operations:

- GET (listing)
- GET (lookup)
- POST
- PUT
- PATCH
- DELETE